lib/EnsEMBL/REST/Controller/Homology.pm

```diff
@@ -130,42 +177,29 @@ sub get_orthologs : Args(0) ActionClass('REST') {
   
   my @final_homologies;
   
+  my $species = $s->{species};  # species is assumed to have been stashed
+  my $gdb = try {
+    $s->{genome_adaptor}->fetch_by_name_assembly( $species );
+  } catch {
+    $c->log->warn("Could not fetch GenomeDB for species [$species], will try aliases");
+  };
+
+  if (!defined $gdb) {
+    my $resolved_species_name = $c->model('Registry')->get_alias( $species );
+    $gdb = try {
+      $s->{genome_adaptor}->fetch_by_name_assembly( $resolved_species_name );
+    } catch {
+      $c->log->error(qq{Species not found in db: $species});
+      $c->go('ReturnError', 'from_ensembl', [qq{$_}]) if $_ =~ /STACK/;
+      $c->go('ReturnError', 'custom', [qq{$_}]);
+    }
+  };
```

Comment:
```suggestion
  }
```

Reply:
Thanks for pointing that out, @ens-sb. I've fixed it now.